homeassistant/components/homekit/strings.json

```diff
@@ -0,0 +1,54 @@
+{
+   "title" : "HomeKit Bridge",
```

Comment:
Let's drop this so it will default to the manifest name (HomeKit)

Reply:
Removed the title.  and changed `manifest` to add `Bridge`.  I'm hoping this clears up the al the "you really want HomeKit Controller" confusion.

Reply:
Oh well in that case we might need to put it back as title, because we need to translate the word bridge.